apps/aecore/src/aec_parent_connector.erl

```diff
@@ -8,15 +8,6 @@
 
 -module(aec_parent_connector).
 
-%% Functionality:
```

Comment:
You deleted all functionality, does that mean that the remaining code has no functionality at all?